nixos/modules/virtualisation/parallels-guest.nix

```diff
@@ -89,5 +89,58 @@ in
       };
     };
 
+    systemd.services.prlcc = {
+      description = "Parallels Control Center";
+      wantedBy = [ "graphical.target" ];
+      serviceConfig = {
+        ExecStart = "${prl-tools}/bin/prlcc";
+      };
+    };
+  
+    systemd.user.services = {
+      prlcc = {
+        description = "Parallels Control Center";
+        wantedBy = [ "graphical-session.target" ];
+        serviceConfig = {
+          ExecStart = "${prl-tools}/bin/prlcc";
```

Comment:
I ask because there was a service called `prldhd` before.

Reply:
Above comment:
>I don't recall seeing prldhd in prl-12, so I'm assuming they merged its functionality into prlcc

I haven't looked at this in a while since I just needed it to work on my system at the time; if I have a chance this weekend I'll get more information for you.